### ISSUE TITLE
Issue running train()

### ISSUE THREAD

**Issue by mehrnoushmalek** [mikeyEcology/MLWIC2#1]
Hello @mikeyEcology. I installed this package, and tried to use `MLWIC2::train()`, but I get the error of `sh: permission denied` for python_loc path.  I don't have this problem when I run `MLWIC::train()`.
I tried to do the setup again but it fails.
`UnsatisfiableError: The following specifications were found
  to be incompatible with the existing python installation in your environment:
  Specifications: - tensorflow -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|3.7.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6, <3.7.0a0|>=3.7,<3.8.0a0]`

`Your python: python=3.8`
And I have Python 3.7, which is used by MLWIC. So I don't know what I'm doing wrong that I cannot make it to work.

**Comment by mikeyEcology** [comment]
What version of tensorflow do you have installed? It looks like you have Python3.8 installed on your computer as well and currently [tensorflow can't be installed on Python 3.8](https://github.com/tensorflow/tensorflow/issues/33374). Can you try downgrading your python to 3.7 for now?

**Comment by mehrnoushmalek** [comment]
When I checked , I only have 3.7 in my anaconda. I searched a bit and it might be due to the fact that I'm using it for MLWIC, cause I checked the ownerhip of the python folder and it was fine. The fact is that it works on MLWIC, but I get the `permission denied` for MLWIC2, so I may need to uninstall it in order to use the second version? I'm more an R person, and python throws so many warning and error that I give up.
